WIP 17254 - added thread naming pattern for the executor service

SivaTharun opened the pull request:
- used guava utils, to create threadfactory, set thread name convention and pass as parameter as executorservice

glefloch committed:
Handle conditional dependency resolution in Gradle

Signed-off-by: Guillaume Le Floch <glfloch@gmail.com>

loicmathieu committed:
Return an Uni<Entity> for persist/update/persistAndFlush methods

Fixes #14475

michalszynkiewicz committed:
gRPC Dev UI - support streaming calls

cescoffier committed:
Implement Dev Services for Kafka

glefloch committed:
Add mailer integration tests

essobedo committed:
Make the maven build task compatible with parallel builds

aloubyansky committed:
Catch more deployment artifacts on the runtime CP

ebullient committed:
CLI 2.x: fix property propagation

gsmet committed:
Fix generated configuration include for Continuous Testing guide

ebullient committed:
Update cli docs

FroMage committed:
Revert "RESTEasy Reactive: allow, test and document HttpServerResponse injection"

This reverts commit 5e4e822f27f2b4aca24faac7cf91d02a793309d9.

mkouba committed:
gRPC Dev UI - init service health statuses after hotdeploy

- plus minor cleanup
- resolves #17453

kenfinnigan committed:
Restructure OpenTelemetry extensions to use a parent aggregator

FroMage committed:
Revert "New APT processor to support validation of annotated methods"

This reverts commit 413c2b6643a505086b289d25263a95eae89d2591.

SivaTharun committed:
17254 - added thread naming pattern for the executor service

- used guava utils, to create threadfactory, set thread name convention and pass as parameter as executorservice

SivaTharun committed:
17254 - added thread naming pattern for the executor service

- used guava utils, to create threadfactory, set thread name convention and pass as parameter as executorservice

SivaTharun committed:
Merge branch 'main' of github.com:quarkusio/quarkus into main

SivaTharun commented:
PR for addressing #17254

gsmet committed:
Merge pull request #17500 from gsmet/fix-generated-doc-ct

Fix generated configuration include for Continuous Testing guide

gsmet committed:
Merge pull request #17502 from ebullient/cli2

CLI v2.x docs (part 2)

gsmet reviewed (changes requested):
Thanks for this.

Could you adjust to not depend on Guava? We don't want a hard dependency to it.

Thanks!

dependabot[bot] committed:
Bump awssdk.version from 2.16.71 to 2.16.73

Bumps `awssdk.version` from 2.16.71 to 2.16.73.

Updates `software.amazon.awssdk:bom` from 2.16.71 to 2.16.73
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.16.71...2.16.73)

Updates `apache-client` from 2.16.71 to 2.16.73

Signed-off-by: dependabot[bot] <support@github.com>

kenfinnigan committed:
Merge pull request #17505 from kenfinnigan/opentelemetry-restructure

Restructure OpenTelemetry extensions to use a parent aggregator

loicmathieu committed:
Merge pull request #17004 from loicmathieu/feat/reactive-panache-persist

Return an Uni<Entity> for persist/update/persistAndFlush methods

cescoffier committed:
Update to Vert.x 4.1.0.CR2

geoand committed:
Use proper parent first artifacts set in fast-jar

Fixes: #17512

famod committed:
Fix EA JDK build via jdk.compiler add-opens

skattela committed:
Add ability to use Pact version >=4.1.0

Relates to quarkusio#9677

famod committed:
Update gitflow-incremental-builder to 3.14.2

mkouba committed:
Merge pull request #17504 from mkouba/issue-17453

gRPC Dev UI - init service health statuses after hotdeploy

cescoffier committed:
Exclude flaky test in the Reactive Messaging TCK - the tests needs to be rewritten in the TCK

aloubyansky committed:
Merge pull request #17484 from essobedo/maven-build-thread-safe

Make the maven build task compatible with parallel builds

gsmet committed:
Merge pull request #17517 from cescoffier/update-to-vertx-4.1.0.CR2

Update to Vert.x 4.1.0.CR2

gsmet committed:
Merge pull request #17510 from quarkusio/dependabot/maven/awssdk.version-2.16.73

Bump awssdk.version from 2.16.71 to 2.16.73

gsmet committed:
Merge pull request #17520 from famod/fix-ea-jdk-build

Fix EA JDK build via jdk.compiler add-opens

famod committed:
Unify NativeBuildReport usage via jbang in GH workflows

dependabot[bot] committed:
Bump awssdk.version from 2.16.73 to 2.16.74

Bumps `awssdk.version` from 2.16.73 to 2.16.74.

Updates `software.amazon.awssdk:bom` from 2.16.73 to 2.16.74
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.16.73...2.16.74)

Updates `apache-client` from 2.16.73 to 2.16.74

Signed-off-by: dependabot[bot] <support@github.com>

Sanne committed:
Remove support for hibernate.properties (long deprecated)

gastaldi committed:
Merge pull request #17524 from famod/gib-3.14.2-update

Update gitflow-incremental-builder to 3.14.2

ebullient committed:
@UnlessBuildProperty to suppress default instance

Co-authored-by: Martin Kouba <mkouba@redhat.com>
Co-authored-by: Stuart Douglas <stuart.w.douglas@gmail.com>

aloubyansky committed:
Introduces a platform release index allowing platform releases be ordered according to the preferences provided by registries

geoand committed:
Merge pull request #17542 from glefloch/add-mailer-tests

Add mailer tests

geoand committed:
Merge pull request #17521 from skattela/main

Add ability to use Pact version >=4.1.0

aloubyansky committed:
Merge pull request #17518 from geoand/#17512

Use proper parent first artifacts Set in fast-jar

michalszynkiewicz committed:
gRPC: fixed running client calls on event loop if needed

mkouba committed:
ArC - do not activate request context for Inialized/Destroyed events

gsmet committed:
Merge pull request #17503 from FroMage/safer-annotation-fix

Safer annotations removal

geoand committed:
Ensure that VertxInputStream doesn't result in infinite attempts to read data

This change is needed because Netty now calls available to determine
whether or not more bytes are available, before calling read.
The negative value that we used to use does not play well with
PushbackInputStream which is used by Netty to wrap the stream.
For a full description of how I can came to this, see
https://github.com/quarkusio/quarkus/issues/17506#issuecomment-851365258

Fixes: #17506

essobedo committed:
Add some doc regarding the new features in the maven plugin
Co-authored-by: Alexey Loubyansky <olubyans@redhat.com>

geoand committed:
Allow extensions to suppress build time config change warnings

Fixes: #17519

mkouba committed:
gRPC - fix a bug in service validation

- fail the build if a BindableService bean is not annotated with
@GrpcService even if the bean was removed
- also fail the build if a bean that implements a generated Mutiny
service interface is not annotated with @GrpcService

geoand committed:
Don't write error response when it's already been written

Fixes: #17548

geoand committed:
Ensure that Spring Cloud Config Server can be reached when port is not set

Fixes: #17522

aloubyansky committed:
Merge pull request #17541 from aloubyansky/new-platform-model

Devtools: introduces a platform release index allowing proper platform releases ordering when selecting BOMs for import

michalszynkiewicz committed:
Merge pull request #17442 from michalszynkiewicz/grpc-devui-streams

gRPC Dev UI - support streaming calls

mkouba committed:
Merge pull request #17550 from mkouba/grpc-warning

gRPC - fix a bug in service validation

geoand committed:
Make VertxInputStream#available not fail on large content-type

An InputStream cannot read over 2GB of memory, but as `available`
is only an estimate, there is no good reason for that specific
method to fail if content-type indicated a larger payload.
Whether or not the processing fails or how it proceeds, should
be up to consumer of the stream and will likely have to do with
how `read` is implemented.

Fixes: #17455

phillip-kruger committed:
SmallRye GraphQL Client: remove hard dependency on Rest client

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

sberyozkin committed:
Merge pull request #17551 from geoand/#17548

Don't write error response when it's already been written

geoand committed:
Merge pull request #17549 from geoand/#17519

Allow extensions to suppress build time config change warnings

ebullient committed:
Update quarkus-bot.yml

Swap Phillip for Ken re: micrometer

geoand committed:
Merge pull request #17553 from geoand/#17522

Ensure that Spring Cloud Config Server can be reached when port is not set

gsmet committed:
Merge pull request #17561 from quarkusio/ebullient-patch-1

Update quarkus-bot.yml

gsmet committed:
Merge pull request #17528 from famod/unify-jbang-report

Unify NativeBuildReport usage via jbang in GH workflows

gsmet committed:
Merge pull request #17498 from aloubyansky/deployments-on-rt-cp

Catch more deployment artifacts on the runtime CP

ebullient committed:
CLI tests: prereqs and property propagation

phillip-kruger committed:
Updated SmallRye GraphQL to 1.2.3

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

geoand committed:
Merge pull request #17555 from geoand/#17455

Make VertxInputStream#available not fail on large content-type

gsmet committed:
Merge pull request #17529 from quarkusio/dependabot/maven/awssdk.version-2.16.74

Bump awssdk.version from 2.16.73 to 2.16.74

gsmet committed:
Merge pull request #17567 from ebullient/cli-tests

CLI tests: prereqs and property propagation

gsmet committed:
Merge pull request #17556 from phillip-kruger/smallrye-graphql-client-deps

SmallRye GraphQL Client: remove hard dependency on Rest client

ebullient committed:
Propagate environment for CLI tests

stuartwdouglas committed:
Continuous tests may not run for unit tests

Fixes #17557

stuartwdouglas committed:
Create generic build item for DevServices native

michalszynkiewicz committed:
gRPC: support @Blocking on class

fixes #17475

jmartisk committed:
Merge pull request #17554 from phillip-kruger/smallrye-graphql-1.2.3

Updated SmallRye GraphQL to 1.2.3

stuartwdouglas committed:
Merge pull request #17546 from geoand/#17506

Ensure that VertxInputStream doesn't result in infinite attempts to read data

phillip-kruger committed:
Upgrade SmallRye OpenAPI to 2.1.5

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

michalszynkiewicz committed:
gRPC: fail code generation gracefully on default java package

gsmet committed:
Merge pull request #17572 from ebullient/jbang-cli

Propagate environment for CLI tests

gsmet committed:
Merge pull request #17547 from essobedo/doc-maven-multi-build

Add some doc regarding the new features in the maven plugin

geoand committed:
Create log file parent directories for @QuarkusIntegrationTest

Fixes: #17565

mkouba committed:
Merge pull request #17544 from mkouba/arc-reqcontext-lifecycle-events

ArC - do not activate request context for Inialized/Destroyed events

stuartwdouglas committed:
Merge pull request #17573 from stuartwdouglas/17557

Continuous tests may not run for unit tests

sberyozkin committed:
Update to SmallRye JWT 3.2.0

aloubyansky committed:
Use project's Quarkus version when resolving extension catalogs using a registry client

cescoffier committed:
Merge pull request #17577 from michalszynkiewicz/grpc-fail-on-default-package

cescoffier committed:
Merge pull request #17574 from stuartwdouglas/generic-dev-services-build-item

gsmet committed:
Merge pull request #17576 from phillip-kruger/smallrye-openapi-2.1.5

Upgrade SmallRye OpenAPI to 2.1.5

jmartisk committed:
Update to GraphQL client user guide

mkouba committed:
Merge pull request #17570 from ebullient/vetoDefaultUnless

@UnlessBuildProperty to suppress default instance

geoand committed:
Apply tiny polish to DevMojo

manusa committed:
Bump kubernetes-client-bom from 5.4.0 to 5.4.1

Signed-off-by: Marc Nuri <marc@marcnuri.com>

glefloch committed:
Bump gradle jandex plugin version in test and doc

aloubyansky committed:
Merge pull request #17585 from aloubyansky/project-quarkus-version-registry-client

Use project's Quarkus version when resolving extension catalogs using a registry client

geoand committed:
Preserve TCCL when using coroutines in RESTEasy Reactive

Fixes: #17597

aloubyansky committed:
Merge pull request #16147 from glefloch/fix/gradle-conditional-dependencies

Handle conditional dependency resolution in Gradle

sberyozkin committed:
Merge pull request #17584 from sberyozkin/smallrye-jwt-3.2.0

Update to SmallRye JWT 3.2.0

gsmet committed:
Do not delete BrotliDecoder and let GraalVM DCE do it for us

gsmet committed:
Merge pull request #17599 from glefloch/bump/gradle-jandex-plugin

Bump gradle jandex plugin version in test and doc

yrodiere committed:
Upgrade to Jandex 2.3.0

gsmet committed:
Pass max frame size to VertxServerWebSocketContainer

Hopefully fixes #17206

yrodiere committed:
Upgrade to jandex-maven-plugin 1.1.0

yrodiere committed:
Upgrade to org.kordamp.gradle.jandex 0.11.0

kenfinnigan committed:
Resolves #17385

- Moves REST Client filter into a separate hidden extension
- Uses Quarkus conditional dependency handling to only include the OpenTelemetry Rest Client extension if `quarkus-rest-client` is present in user application

Sanne committed:
Upgrade to Hibernate ORM 5.5.0.Final

ebullient committed:
Use main.java for JBang projects

gsmet committed:
Merge pull request #17598 from manusa/deps/kubernetes-client-5.4.1

Bump kubernetes-client-bom from 5.4.0 to 5.4.1

Sanne committed:
Upgrade to Hibernate Reactive 1.0.0.CR6

Sanne committed:
Fail deployment on presence of an hibernate.properties resource

gsmet committed:
Merge pull request #17609 from Sanne/ORMUpgrade

Upgrade to Hibernate ORM 5.5.0.Final

dependabot[bot] committed:
Bump awssdk.version from 2.16.74 to 2.16.75

Bumps `awssdk.version` from 2.16.74 to 2.16.75.

Updates `software.amazon.awssdk:bom` from 2.16.74 to 2.16.75
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.16.74...2.16.75)

Updates `apache-client` from 2.16.74 to 2.16.75

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

kenfinnigan committed:
Merge pull request #17608 from kenfinnigan/otel-rest-client

stuartwdouglas committed:
Merge pull request #17580 from geoand/#17565

Create log file parent directories for @QuarkusIntegrationTest

stuartwdouglas committed:
Merge pull request #17605 from gsmet/maxframesize-websockets

Pass max frame size to VertxServerWebSocketContainer

stuartwdouglas committed:
Don't run maven IT names by default

This is not perfect, as this information
should be read from the plugin config,
but it will prevent many issues.

Relates to #17558

stuartwdouglas committed:
Merge pull request #17601 from geoand/#17597

Preserve TCCL when using coroutines in RESTEasy Reactive

yrodiere committed:
Revert part of "Hibernate Validator - Work around Jandex issue in enclosingTarget()"

This reverts the workaround from commit f6cffe0efdf4215ce09da17d801265905075e4ef,
but preserves the test.

Assuming jandex indexes are always built with Jandex 2.3.0+, we no
longer need this workaround.

michalszynkiewicz committed:
Merge pull request #17575 from michalszynkiewicz/grpc-blocking-on-class

gRPC: support @Blocking on class

cescoffier committed:
Update Vert.x to version 4.1.0
Update Vert.x Mutiny bindings to version 2.6.0
Update SmallRye Reactive Messaging to version: 3.4.0

cescoffier committed:
Merge pull request #17617 from stuartwdouglas/17558

Don't run maven IT names by default

gsmet committed:
Merge pull request #17604 from gsmet/brotli-dce

Do not delete BrotliDecoder and let GraalVM DCE do it for us

geoand committed:
Merge pull request #17596 from geoand/devmojo-tiny-polish

Apply tiny polish to DevMojo

Sgitario committed:
CLI: Change default short option to display version from "-V" to "-v"

The current short option to display the version is "-V" (uppercase): `quarkus -V`. This is set by Picocli, but can be changed using the `picocli.version.name.0` property.

The problem with `-V` is that does not match with the other short options to display version in the extensions and other commands which is "-v".
Also, in other command tools like `mvn`, the option is `-v`.
I did check that the `-v` is not conflicting with other existing options.

gsmet committed:
Merge pull request #17615 from quarkusio/dependabot/maven/awssdk.version-2.16.75

Bump awssdk.version from 2.16.74 to 2.16.75

geoand committed:
Ensure @QuarkusIntegrationTest works properly with any Filesystem path

Fixes: #17607

geoand committed:
Add rule for @QuarkusIntegrationTest to the bot

Sanne committed:
Merge pull request #17538 from Sanne/NoMoreHibernateProperties

Remove support for hibernate.properties (long deprecated)

gsmet committed:
Merge pull request #17620 from cescoffier/update-vertx-reactive-messaging-mutiny-bindings

Update Vert.x to version 4.1.0 and related dependencies

gsmet committed:
Merge pull request #17624 from geoand/bot-quarkusintegrationtest

Add rule for @QuarkusIntegrationTest to the bot

gsmet committed:
Merge pull request #17613 from Sanne/HRUpgrade

Upgrade to Hibernate Reactive 1.0.0.CR6

geoand committed:
Merge pull request #17623 from geoand/#17607

Ensure @QuarkusIntegrationTest works properly with any Filesystem path

gsmet committed:
Correctly propagate dispatchToWorker to websockets

Also use the executor included in the container info even if the code
was working ok.

Fixes #17571

gsmet committed:
Merge pull request #17590 from jmartisk/update-graphql-client-docs

Update to GraphQL client user guide

jmartisk committed:
Add the smallrye-graphql-client extension to MAINTAINERS.adoc

kenfinnigan committed:
Remove default Jaeger endpoint

Resolves #16107

- Remove the default Jaeger endpoint from configuration
- When in DEV mode, default the endpoint to `http://localhost:14250` if it's not set

michalszynkiewicz committed:
gRPC: IOThreadInterceptor check if context != null before delegating to event loop

geoand committed:
Add support for @Cache and @NoCache in RESTEasy Reactive

Fixes: #17630

gsmet committed:
Merge pull request #17635 from jmartisk/update-maintainers

Add the smallrye-graphql-client extension to MAINTAINERS.adoc

kenfinnigan committed:
Merge pull request #17634 from kenfinnigan/otel-jaeger-endpoint

Remove default Jaeger endpoint

gsmet committed:
Merge pull request #17636 from michalszynkiewicz/grpc-blocking-context

gRPC: IOThreadInterceptor must check if context is not null before calling runOnContext

gsmet committed:
Merge pull request #17621 from Sgitario/cli_change_version_param

CLI: Change default short option to display version from "-V" to "-v"

kenfinnigan committed:
Add Guide for OpenTelemetry

Resolves #16093

glefloch committed:
Export maven.repo.local for gradle tests

famod committed:
CI: cancel also non-PR runs in forks

dependabot[bot] committed:
Bump aws-alexa-sdk.version from 2.38.1 to 2.39.0

Bumps `aws-alexa-sdk.version` from 2.38.1 to 2.39.0.

Updates `ask-sdk` from 2.38.1 to 2.39.0
- [Release notes](https://github.com/amzn/alexa-skills-kit-java/releases)
- [Commits](https://github.com/amzn/alexa-skills-kit-java/commits)

Updates `ask-sdk-apache-client` from 2.38.1 to 2.39.0
- [Release notes](https://github.com/amzn/alexa-skills-kit-java/releases)
- [Commits](https://github.com/amzn/alexa-skills-kit-java/commits)

---
updated-dependencies:
- dependency-name: com.amazon.alexa:ask-sdk
  dependency-type: direct:production
  update-type: version-update:semver-minor
- dependency-name: com.amazon.alexa:ask-sdk-apache-client
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

stuartwdouglas committed:
Change the way Resteasy injection is initialized

Fixes #17628

stuartwdouglas committed:
Merge pull request #17632 from gsmet/websocket-dispatch-to-worker

Correctly propagate dispatchToWorker to websockets

jmartisk committed:
Disable GraphQL metrics by default

geoand committed:
Merge pull request #17644 from geoand/#17630

Add support for @Cache and @NoCache in RESTEasy Reactive

jmartisk committed:
Rename quarkus.neo4j.pool.metrics-enabled for unification

yrodiere committed:
Merge pull request #17645 from famod/ci-concurrency-forks

CI: cancel also non-PR runs in forks

mkouba committed:
Merge pull request #17654 from jmartisk/issue-17448-neo4j

Rename quarkus.neo4j.pool.metrics-enabled for unification

geoand committed:
Prevent double registration of MpPublisherMessageBodyReader in rest client

Fixes: #17627

geoand committed:
Merge pull request #17651 from stuartwdouglas/17628

Change the way Resteasy injection is initialized

gsmet committed:
Merge pull request #17648 from quarkusio/dependabot/maven/aws-alexa-sdk.version-2.39.0

Bump aws-alexa-sdk.version from 2.38.1 to 2.39.0

dependabot[bot] committed:
Bump awssdk.version from 2.16.75 to 2.16.76

Bumps `awssdk.version` from 2.16.75 to 2.16.76.

Updates `software.amazon.awssdk:bom` from 2.16.75 to 2.16.76
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.16.75...2.16.76)

Updates `apache-client` from 2.16.75 to 2.16.76

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

geoand committed:
Allow specifying per-port nodePort

Resolves: #17582

gsmet committed:
Initialize io.vertx.ext.auth.impl.jose.JWT at runtime

It contains a Random field so we can't initialize it at static init.

Fixes #17657

geoand committed:
Add documentation for @Cache and @NoCache

Follows up on #17644

Co-authored-by: Stéphane Épardaud <stef@inforealm.org>

gsmet committed:
Initialize io.grpc.internal.RetriableStream at runtime

It contains a Random field.

Fixes #17234

geoand committed:
Merge pull request #17656 from geoand/#17627

Prevent double registration of MpPublisherMessageBodyReader in rest client

gsmet committed:
Merge pull request #17639 from kenfinnigan/guide-otel

Add Guide for OpenTelemetry

cescoffier committed:
Merge pull request #17468 from cescoffier/dev-services-for-kafka

jmartisk committed:
Merge pull request #17652 from jmartisk/issue-17448-graphql

Disable GraphQL metrics by default

geoand committed:
Merge pull request #17663 from gsmet/grpc-runtime

Initialize io.grpc.internal.RetriableStream at runtime

geoand committed:
Merge pull request #17661 from geoand/rr-cache-doc

Add documentation for @Cache and @NoCache

gsmet committed:
Merge pull request #17643 from glefloch/fix/release-test

Export maven.repo.local for gradle tests

gsmet committed:
Merge pull request #17660 from gsmet/jwt-runtime

Initialize io.vertx.ext.auth.impl.jose.JWT at runtime

gsmet committed:
Merge pull request #17646 from quarkusio/dependabot/maven/awssdk.version-2.16.76

Bump awssdk.version from 2.16.75 to 2.16.76

mkouba committed:
Make sure the gRPC server is started after synthetic beans init

gsmet committed:
Upgrade to SmallRye GraphQL 1.2.4

mkouba committed:
ArC - document io.quarkus.arc.WithCaching

- resolves #17625

cescoffier committed:
Start rewriting the reactive documentation

gsmet committed:
Merge pull request #17669 from mkouba/grpc-start-after-synthetic-beans

Make sure the gRPC server is started after synthetic beans init

gsmet committed:
Merge pull request #17606 from yrodiere/jandex2.3

Upgrade to Jandex 2.3.0

dependabot[bot] committed:
Bump mockito-core from 3.10.0 to 3.11.0 in /integration-tests/gradle

Bumps [mockito-core](https://github.com/mockito/mockito) from 3.10.0 to 3.11.0.
- [Release notes](https://github.com/mockito/mockito/releases)
- [Commits](https://github.com/mockito/mockito/compare/v3.10.0...v3.11.0)

---
updated-dependencies:
- dependency-name: org.mockito:mockito-core
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump mockito-core from 3.10.0 to 3.11.0 in /devtools/gradle

Bumps [mockito-core](https://github.com/mockito/mockito) from 3.10.0 to 3.11.0.
- [Release notes](https://github.com/mockito/mockito/releases)
- [Commits](https://github.com/mockito/mockito/compare/v3.10.0...v3.11.0)

---
updated-dependencies:
- dependency-name: org.mockito:mockito-core
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

ebullient committed:
Merge pull request #17610 from ebullient/jbang-main

Use main.java for JBang projects

dependabot[bot] committed:
Bump aws-xray-recorder-sdk-aws-sdk-v2 from 2.9.0 to 2.9.1

Bumps [aws-xray-recorder-sdk-aws-sdk-v2](https://github.com/aws/aws-xray-sdk-java) from 2.9.0 to 2.9.1.
- [Release notes](https://github.com/aws/aws-xray-sdk-java/releases)
- [Changelog](https://github.com/aws/aws-xray-sdk-java/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-xray-sdk-java/compare/v2.9.0...v2.9.1)

---
updated-dependencies:
- dependency-name: com.amazonaws:aws-xray-recorder-sdk-aws-sdk-v2
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump mockito-core from 3.10.0 to 3.11.0

Bumps [mockito-core](https://github.com/mockito/mockito) from 3.10.0 to 3.11.0.
- [Release notes](https://github.com/mockito/mockito/releases)
- [Commits](https://github.com/mockito/mockito/compare/v3.10.0...v3.11.0)

---
updated-dependencies:
- dependency-name: org.mockito:mockito-core
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump flyway-core from 7.9.1 to 7.9.2

Bumps [flyway-core](https://github.com/flyway/flyway) from 7.9.1 to 7.9.2.
- [Release notes](https://github.com/flyway/flyway/releases)
- [Commits](https://github.com/flyway/flyway/compare/flyway-7.9.1...flyway-7.9.2)

---
updated-dependencies:
- dependency-name: org.flywaydb:flyway-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

cescoffier committed:
Merge pull request #17675 from cescoffier/reactive-doc-part-1

Start rewriting the reactive documentation

stuartwdouglas committed:
Capabilities are not persisted

Fixes #17592

geoand committed:
Use @Identifier instead of @Named in kafka client

This is needed to remove a warning from SR RM

gsmet committed:
Merge pull request #17679 from quarkusio/dependabot/gradle/integration-tests/gradle/org.mockito-mockito-core-3.11.0

Bump mockito-core from 3.10.0 to 3.11.0 in /integration-tests/gradle

gsmet committed:
Merge pull request #17680 from quarkusio/dependabot/gradle/devtools/gradle/org.mockito-mockito-core-3.11.0

Bump mockito-core from 3.10.0 to 3.11.0 in /devtools/gradle

gsmet committed:
Merge pull request #17681 from quarkusio/dependabot/maven/com.amazonaws-aws-xray-recorder-sdk-aws-sdk-v2-2.9.1

Bump aws-xray-recorder-sdk-aws-sdk-v2 from 2.9.0 to 2.9.1

gsmet committed:
Merge pull request #17682 from quarkusio/dependabot/maven/org.mockito-mockito-core-3.11.0

Bump mockito-core from 3.10.0 to 3.11.0

gsmet committed:
Merge pull request #17683 from quarkusio/dependabot/maven/org.flywaydb-flyway-core-7.9.2

Bump flyway-core from 7.9.1 to 7.9.2

gsmet committed:
Upgrade Quarkus HTTP to 4.1.1

mkouba committed:
Merge pull request #17671 from mkouba/issue-17625

ArC - document io.quarkus.arc.WithCaching

geoand committed:
Merge pull request #17690 from geoand/kafka-identifier

Use @Identifier instead of @Named in kafka client

stuartwdouglas committed:
Merge pull request #17689 from stuartwdouglas/17592

Capabilities are not persisted

gsmet committed:
Merge pull request #17692 from gsmet/quarkus-http-4.1.1

Upgrade Quarkus HTTP to 4.1.1

geoand committed:
Merge pull request #17659 from geoand/#17582

Allow specifying per-port nodePort

geoand committed:
Remove unused build mongo dev-service build items

dependabot[bot] committed:
Bump aws-lambda-java-events from 3.8.0 to 3.9.0

Bumps [aws-lambda-java-events](https://github.com/aws/aws-lambda-java-libs) from 3.8.0 to 3.9.0.
- [Release notes](https://github.com/aws/aws-lambda-java-libs/releases)
- [Commits](https://github.com/aws/aws-lambda-java-libs/commits)

---
updated-dependencies:
- dependency-name: com.amazonaws:aws-lambda-java-events
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump neo4j-java-driver from 4.3.0 to 4.3.1

Bumps [neo4j-java-driver](https://github.com/neo4j/neo4j-java-driver) from 4.3.0 to 4.3.1.
- [Release notes](https://github.com/neo4j/neo4j-java-driver/releases)
- [Commits](https://github.com/neo4j/neo4j-java-driver/compare/4.3.0...4.3.1)

---
updated-dependencies:
- dependency-name: org.neo4j.driver:neo4j-java-driver
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump awssdk.version from 2.16.76 to 2.16.78

Bumps `awssdk.version` from 2.16.76 to 2.16.78.

Updates `software.amazon.awssdk:bom` from 2.16.76 to 2.16.78
- [Release notes](https://github.com/aws/aws-sdk-java-v2/releases)
- [Changelog](https://github.com/aws/aws-sdk-java-v2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-java-v2/compare/2.16.76...2.16.78)

Updates `apache-client` from 2.16.76 to 2.16.78

---
updated-dependencies:
- dependency-name: software.amazon.awssdk:bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: software.amazon.awssdk:apache-client
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Merge pull request #17709 from quarkusio/dependabot/maven/awssdk.version-2.16.78

dependabot[bot] committed:
Merge pull request #17707 from quarkusio/dependabot/maven/com.amazonaws-aws-lambda-java-events-3.9.0

dependabot[bot] committed:
Merge pull request #17708 from quarkusio/dependabot/maven/org.neo4j.driver-neo4j-java-driver-4.3.1

machi1990 committed:
Merge pull request #17705 from geoand/mongo-old-build-item

Remove unused build mongo dev-service build items

SivaTharun committed:
Merge branch 'main' of github.com:SivaTharun/quarkus
